portal.php

```diff
@@ -293,6 +283,16 @@
 				}
 			}
 		}
+		elseif(my_strpos($user['sid'], 'bot=') !== false && $spiders[$botkey])
+		{
+			// The user is a search bot.
+			$onlinebots[] = format_name($spiders[$botkey]['name'], $spiders[$botkey]['usergroup']);
```

Comment:
Need to use explicit key while including bots in array, inline with @euantorano 's PR